### ISSUE TITLE
Positioning wind roses onto an existing x-y axis

### ISSUE THREAD

**Issue by guidocioni** [python-windrose/windrose#188]
I have a meteogram with many variables and I would like to add a "strip" with wind roses like this

<img width="849" alt="Screen Shot 2022-09-12 at 16 34 46" src="https://user-images.githubusercontent.com/12760310/189682052-fb455818-90db-4986-a9d5-6228f2c7c34f.png">

I'm using gridspec and already made space for an axis (`ax_winds`). Following the example on the documentation I'm trying to create an inset ax based on this axis and just tried to plot a single wind rose at a certain point in time.

```python
gs = gridspec.GridSpec(nrows=4, ncols=1, height_ratios=[1, 1, 0.3, 1])
ax_winds = plt.subplot(gs[2])
wrax_cham = inset_axes(ax_winds,
         width=1,                             # size in inches
         height=1,                            # size in inches
         loc='center',                        # center bbox at given position
         bbox_to_anchor=(dset_city['valid_time'][5], 0.5), # position of the axe
         bbox_transform=ax_winds.transData,    # use data coordinate (not axe coordinate)
         axes_class=WindroseAxes,    # specify the class of the axe
)
wrax_cham.bar(wdir[:,5], wspd[:,5], opening=1, edgecolor='white')
```

But I'm getting some errors 

```python
ValueError: 'bboxes' cannot be empty
```

which are probably related to how I'm passing the parent axis to `inset_axes`. 

Does anyone have an example of positioning multiple wind roses over a define time axis? 

Full code is here if someone is interested https://github.com/guidocioni/icon_d2_eps/blob/master/plotting/plot_meteogram.py

**Comment by ocefpaf** [comment]
What happens if you pass the projection in your subplot?

```
plt.subplot(gs[2],  projection"="windrose")
```

**Comment by guidocioni** [comment]
> projection"="windrose"

Yep, that fixes the issue. However now I have two different wind roses appearing...

![bfbde44d-5e30-4baa-9b76-5f3cde5b565c](https://user-images.githubusercontent.com/12760310/192089518-2d46df69-a71d-4834-bfe6-f039c87496f3.png)

Any idea what's going on? For now I'm just plotting a single time step

```python
ax_winds = plt.subplot(gs[2], projection="windrose")
wspd = wind_speed(dset_city['u10'], dset_city['v10']).metpy.dequantify()
wdir = wind_direction(dset_city['u10'], dset_city['v10']).metpy.dequantify()
wrax_cham = inset_axes(ax_winds,
         width=0.8,                             # size in inches
         height=0.8,                            # size in inches
         loc='center',                        # center bbox at given position
         bbox_to_anchor=(dset_city['valid_time'][5], 0.3), # position of the axe
         bbox_transform=ax_winds.transData,    # use data coordinate (not axe coordinate)
         axes_class=WindroseAxes,    # specify the class of the axe
)
wrax_cham.contourf(wdir[:,30], wspd[:,30])
wrax_cham.tick_params(labelleft=False, labelbottom=False)
ax_winds.xaxis.set_major_locator(locator)
ax_winds.xaxis.set_major_formatter(formatter)
ax_winds.tick_params(axis='y', which='major', labelleft=False)
ax_winds.tick_params(axis='x', which='both', labelbottom=False)
```

**Comment by ocefpaf** [comment]
Not sure. If you can make a self-contained example that one can easily run without external data I can try to debug. My guess is that mpl is drawing the polar axis too and not only just the windrose.

**Comment by guidocioni** [comment]
> Not sure. If you can make a self-contained example that one can easily run without external data I can try to debug. My guess is that mpl is drawing the polar axis too and not only just the windrose.

I noticed that if I reduce the size of the axis then it seems to fill the background circle.
Anyhow, I took the time and packed a self-contained example with some test data (you need some data to plot...): it's really small, have a look.

https://drive.google.com/file/d/1SZo0rzTIO6nSDkHVywXg_NeZEX7ptg5L/view?usp=sharing
https://drive.google.com/file/d/14WfhMxCta5xHjkulaznmgxj6lUzt-Rcm/view?usp=sharing

I'm trying to reproduce something similar to this 

<img width="796" alt="Screen Shot 2022-09-26 at 17 27 14" src="https://user-images.githubusercontent.com/12760310/192317867-d6e2ff39-7c6e-4f3d-9fcf-a99b992b71a8.png">

of Course, it doesn't have to be exactly the same.
